src/test/java/seedu/momentum/model/project/ProjectTest.java

```diff
@@ -55,8 +55,9 @@ public void isSameProject() {
     @Test
     public void equals() {
         // same values -> returns true
-        Project aliceCopy = new ProjectBuilder(ALICE).build();
-        assertTrue(ALICE.equals(aliceCopy));
+        Project aliceCopy1 = new ProjectBuilder(ALICE).build();
+        Project aliceCopy2 = new ProjectBuilder(ALICE).build();
+        assertTrue(aliceCopy1.equals(aliceCopy2));
```

Comment:
Can I just check what the reason for changing this was?

Reply:
It was related to the immutability thing I told you about last time. Its not relevant anymore but they're both equivalent